site/.github/bin/deploy_docs.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        -v|--version) ICEBERG_VERSION="$2"; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+GIT_BRANCH="docs-${ICEBERG_VERSION}"
+
+# change to branch
+git checkout -b $GIT_BRANCH
+
+cd ..
+
+# remove all files and directories minus exceptions
+find . \
+ -not \( -path . \) \
+ -not \( -path .. \) \
+ -not \( -path ./.git -prune \) \
```

Comment:
Ah, so this actually runs in the root of the iceberg repo itself?

That's a bit of a problem because we won't want to run it locally. I would recommend updating this to not require deleting anything. Instead, I think this should copy what you need into folders and add just those folders.

To me, the steps should be:
1. Copy the `nightly` docs folder into a temp folder
2. Build the docs in the temp folder
3. Copy the contents needed out of the temp site folder
4. Check out an orphan branch (with no parent) using `--orphan`
5. Add the site folder to git
6. Commit the site folder
7. Switch back to the previous tag

That way, everything in the working copy is handled by git rather than by removing it.